test/unit_tests/augmentation/test_transforms.py

```diff
@@ -631,6 +633,21 @@ def test_mixup_transform(rng_seed, random_batch, alpha, beta_per_sample):
 ]
 
 
+@pytest.mark.parametrize("n_segments", [5, 10, None])
```

Comment:
Please test more, and add more variability for the tests (more functions and parameters).


Include edge cases, n segment 0, 1, ..., number of dots, you need to test with prob 0 and 1, make work with composition with another augmentation and/or pre processing. 

More test also using augmented mode as present in the examples.